presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionInterpreter.java

```diff
@@ -1001,16 +1001,23 @@ protected Object visitLambdaExpression(LambdaExpression node, Object context)
         @Override
         protected Object visitBindExpression(BindExpression node, Object context)
         {
-            Object value = process(node.getValue(), context);
+            List<Object> values = node.getValues().stream()
+                    .map(value -> process(value, context))
+                    .collect(toImmutableList());
             Object function = process(node.getFunction(), context);
 
-            if (hasUnresolvedValue(value, function)) {
+            if (hasUnresolvedValue(values) || hasUnresolvedValue(function)) {
+                ImmutableList.Builder builder = ImmutableList.builder();
```

Comment:
`ImmutableList.Builder<...>`. You should have gotten a warning from IntelliJ when you passed `builder.build()` to `newExpression`.